src/GraphBuilder.js

```diff
@@ -106,7 +106,15 @@ export default class GraphBuilder {
   }
 
   buildLiteralValue(value, subjectTerm, propertyTerm) {
-    const valueTerm = rdf.literal(value.literal, value.lang)
+    const validationDataType =
+      value.property?.propertyTemplate?.validationDataType
```

Comment:
What happens if we change this to `value.property.propertyTemplate.validationDataType`? My concern is that when the property template is missing, we're defaulting to using language, when it is probably an error.

Reply:
I'm not clear on when the propertyTemplate would be missing in real data.   In our test data, removing the `?` gives  things like:

```
Failed test diffs:
 FAIL  /__tests__/actionCreators/resources.newResourceFromDataset.test.js
  ● newResourceFromDataset › loading a resource › dispatches actions

    TypeError: Cannot read properties of undefined (reading 'propertyTemplate')

      108 |   buildLiteralValue(value, subjectTerm, propertyTerm) {
      109 |     const validationDataType =
    > 110 |       value.property.propertyTemplate?.validationDataType
          |                      ^
      111 |     let valueTerm
      112 |     if (validationDataType) {
      113 |       const namedNode = rdf.namedNode(validationDataType)

      at GraphBuilder.buildLiteralValue (src/GraphBuilder.js:110:22)
      at GraphBuilder.buildValue (src/GraphBuilder.js:104:12)
      at forEach (src/GraphBuilder.js:87:14)
          at Array.forEach (<anonymous>)
      at GraphBuilder.buildProperty (src/GraphBuilder.js:86:14)
      at forEach (src/GraphBuilder.js:39:12)
          at Array.forEach (<anonymous>)
      at GraphBuilder.buildSubject (src/GraphBuilder.js:38:24)
      at GraphBuilder.graph (src/GraphBuilder.js:25:12)
      at _callee$ (__tests__/actionCreators/resources.newResourceFromDataset.test.js:82:9)
...
```

And using the new "injectPropertyIntoValue" flag doesn't help:

```
Failed test diffs:
 FAIL  /__tests__/actionCreators/resources.newResourceFromDataset.test.js
  ● Test suite failed to run

    TypeError: Converting circular structure to JSON
        --> starting at object with constructor 'Object'
        |     property 'values' -> object with constructor 'Array'
        |     index 0 -> object with constructor 'Object'
        --- property 'property' closes the circle
        at stringify (<anonymous>)

      at messageParent (node_modules/jest-worker/build/workers/messageParent.js:34:19)
```

And I once agin checked in my browser and am certain we don't have easy access to the `validationDataType` property from any but the `value` argument.   Unless we should be loading up the property from the key ...